core/common/src/main/java/alluxio/underfs/BaseUnderFileSystem.java

```diff
@@ -83,6 +83,11 @@ public AccessControlList getAcl(String path) throws IOException {
     return null;
   }
 
+  @Override
+  public void setAcl(String path, AccessControlList acl) throws IOException {
+    throw new UnsupportedOperationException("setting ACL not supported");
```

Comment:
If acls are not supported, should it it just be a noop? (similar to `getAcl`)

Reply:
done